### PR TITLE
CVSB-4858

### PR DESCRIPTION
**Pull request by mdclarke21**
Also fixes the issue were clicking the test station phone number calls the IT support number.

### CODE REVIEW

File: src/pages/signature-pad/signature-pad.ts [hunk 1]
```diff
@@ -77,7 +80,10 @@ export class SignaturePadPage implements OnInit {
         }, {
           text: APP_STRINGS.CALL_SUPP_BTN,
           handler: () => {
-            console.log('technical support clicked');
+            this.callNumber.callNumber(AppConfig.KEY_PHONE_NUMBER, true).then(
+              data => console.log(data),
```
Comment (adpetrescu) [inline src/pages/signature-pad/signature-pad.ts]: Do we need this `then`?

File: src/pages/test-station/test-station-details/test-station-details.ts [hunk 2]
```diff
@@ -117,11 +117,10 @@ export class TestStationDetailsPage implements OnInit {
         {
           text: APP_STRINGS.CALL,
           handler: () => {
-            this.callNumber.callNumber(AppConfig.KEY_PHONE_NUMBER, true).then(
+            this.callNumber.callNumber(this.testStation.testStationContactNumber, true).then(
               data => console.log(data),
```
Comment (adpetrescu) [inline src/pages/test-station/test-station-details/test-station-details.ts]: same as above

File: src/pages/signature-pad/signature-pad.ts (hunk 1 above)
Comment (mdclarke21) [reply src/pages/signature-pad/signature-pad.ts]: Done

**Comment by mdclarke21** [comment]
Done

File: src/pages/test-station/test-station-details/test-station-details.ts (hunk 2 above)
Comment (mdclarke21) [reply src/pages/test-station/test-station-details/test-station-details.ts]: Done

**Pull request merged**